frontend/talentsearch/src/js/api/workLocationPreferenceOperations.graphql

```diff
@@ -0,0 +1,15 @@
+query WorkLocationPreference{
```

Comment:
I love that this is factored out into its own file.  Could you move this out of `api` and into your `workLocationPreferenceForm` directory?